src/modules/charging-import/lib/import.js

```diff
@@ -30,8 +29,6 @@ const importChargingData = async () => {
   try {
     logger.info('Starting charge data import');
 
-    await chargeVersionImportService.importChargeVersions();
```

Comment:
Please note that I made this same change in my PR. Potential conflict but should be minor, like an extra blank line or something daft like that. No biggie.

Reply:
Done